rules/do-you-label-broken-equipment/rule.md

```diff
@@ -1,38 +1,36 @@
 ---
 type: rule
-archivedreason: 
 title: Office environment - Do you label broken equipment?
-guid: e3a33fc1-ad97-4377-abaf-46f8f52b2edd
 uri: do-you-label-broken-equipment
-created: 2012-09-25T17:51:22.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Ulysses Maclaren
-  url: https://ssw.com.au/people/ulysses-maclaren
-- title: Tiago Araujo
-  url: https://ssw.com.au/people/tiago-araujo
-related: 
-- do-you-fix-problems-quickly
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Ulysses Maclaren
+    url: https://ssw.com.au/people/ulysses-maclaren
+  - title: Tiago Araujo
+    url: https://ssw.com.au/people/tiago-araujo
+related:
+  - do-you-fix-problems-quickly
 redirects:
-- office-environment-do-you-label-broken-equipment
-
+  - office-environment-do-you-label-broken-equipment
+created: 2012-09-25T17:51:22.000Z
+archivedreason: null
+guid: e3a33fc1-ad97-4377-abaf-46f8f52b2edd
 ---
-
 Imagine you see one port of a hub not working, what do you do?
 
-When you see something not working like a phone or a PowerPoint, don't ignore it, stick a note to it and tell the appropriate person.
```

Comment:
I think PowerPoint was correct, see below image
![image](https://user-images.githubusercontent.com/79821522/164362843-ffd5ac53-b4bf-4108-b5a7-d6c15b40bc58.png)

Reply:
I was wrong, it's a literal power point, not MS PowerPoint